packages/cli/src/commands/add.ts

```diff
@@ -21,13 +32,20 @@ async function action(contractNames: string[], options: any): Promise<void> {
   if(!options.skipCompile) await Compiler.call();
   if(options.all) addAll({});
   else {
-    const contractsData = contractNames.map((rawData) => {
-      const [ aName, alias ] = rawData.split(':');
-      return { name: aName, alias: (alias || aName) };
-    });
+    const contractList = Truffle.getContractNames();
+    const promptedArgs = await promptForArgumentsIfNeeded({ args: { contractNames }, props: argsProps(contractList) });
+    const contractsData = contractNames.length !== 0
+      ? contractNames.map(splitContractName)
+      : promptedArgs.contractNames.map(splitContractName);
```

Comment:
Is it necessary to run the result of `promptedArgs` via `split`?

Reply:
Nope, it's not. I  just wanted to maintain consistency with what we had before. As a matter of fact, I noticed that we're also double-checking whether the `alias` property is defined or not (well, not strictly verifying this but checking the `contractNames` type) inside `scripts/add.ts`. I'll probably review why we are doing this (maybe it's just a hotfix to make some tests work?) and remove it if possible.